examples/role-playbook/vg/vgconvert.retry

```diff
@@ -0,0 +1,2 @@
+10.70.42.203
+10.70.42.89
```

Comment:
Yes, .retry extension files are automatically generated. I missed deleting this. I'll add it to gitignore to avoid future problems.